HIAPI/Models/Profile.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Profile.swift
+//  HackIllinois
+//
+//  Created by HackIllinois Team on 2/24/21.
+//  Copyright © 2021 HackIllinois. All rights reserved.
+//  This file is part of the Hackillinois iOS App.
+//  The Hackillinois iOS App is open source software, released under the University of
+//  Illinois/NCSA Open Source License. You should have received a copy of
+//  this license in a file with the distribution.
+//
+import Foundation
+import APIManager
+
+public struct ProfileContainer: Decodable, APIReturnable {
+    public let profiles: [Profile]
+
+    public init(from data: Data) throws {
+        let decoder = JSONDecoder()
+        decoder.dateDecodingStrategy = .secondsSince1970
+        self = try decoder.decode(ProfileContainer.self, from: data)
+    }
+}
+
+public struct Profile: Codable, APIReturnable {
+    internal enum CodingKeys: String, CodingKey {
+        case id
+        case firstName
+        case lastName
+        case points
+        case timezone
+        case info = "Description"
```

Comment:
Will eventually be changed to lowercase "description", will ping when it's updated but currently it's uppercase (this will just break when the casing is changed later in API). No need to make any change right now though.